edu.harvard.i2b2.data/Release_1-7/NewInstall/Metadata/scripts/procedures/oracle/pat_count_visits.sql

```diff
@@ -1,3 +1,8 @@
+-- If your i2b2MetaData schema cant read i2b2Data schema
+-- grant select  on  "&&DataSchemaName"."PATIENT_DIMENSION" to "&&schemaName";
+create synonym PATIENT_DIMENSION for "&&DataSchemaName"."PATIENT_DIMENSION";
```

Comment:
Possibly a dumb Oracle question... where is &&DataSchemaName defined?

Reply:
If the script run through Oracle SQLDevloper pops up a window and ask to define each variable.